default-icon-descriptions.js

```diff
@@ -2044,11 +2084,6 @@ export default {
     id: "icon.title.shoes",
     comment: "Title for shoes icon"
   },
-  shoppingcart: {
```

Comment:
Is this icon removed or something?

Reply:
Previously the descriptions was there but the icon was not. When I added it from the new collection I saw that it looked exactly like "torget-shopping". So I removed all mentions of "shopping-cart" until I clarified what to do here. 

I have just confirmed with Henrik that this new shopping-cart icon should in fact replace the "torget-shopping", but whether or not I can introduce such breaking change is not easy to check at the moment. I tried [code search](https://finn-codesearch.svc.prod.finn.no/search?q=TorgetShopping&num=50) with different variations of "torget shopping" but don't know if I can trust the results 😄 Wdyt I should do?

Reply:
Here I added the new "ShoppingCart" icon: [87e5a63](https://github.com/warp-ds/icons/pull/63/commits/87e5a632d383ce3539819bd49ba29b793c2a45df)
I'm thinking that we could at least filter out TorgetShopping from the [Icons table in the tech-docs](https://warp-ds.github.io/tech-docs/components/icons/). This way we could avoid introducing a breaking change by removing that icon completely from this package, while at the same time we'd limit the use of that icon.